configs/ssg_platform_tools-vfxplatform-eln.yaml

```diff
@@ -0,0 +1,50 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: vfxplatform
+  description: The VFX Reference Platform https://vfxplatform.com/
```

Comment:
Suggest using the following description: The VFX Reference Platform (CY2024 DRAFT) https://vfxplatform.com/ since `CY2024 DRAFT` is met by ELN. We should list the rough standard version we're aiming to comply with in the workload since they are each a bit different.